pkg/cmd/render/render.go

```diff
@@ -362,6 +367,19 @@ func writeCertKeyFiles(outputDir, allSecretName, nodeSecretName string, certData
 	return nil
 }
 
+func getBootstrapScalingStrategy(installConfig map[string]interface{}, delayedHAMarkerFile string) (ceohelpers.BootstrapScalingStrategy, error) {
+	// Delayed HA strategy is set if marker file exists on disk.
+	if _, err := os.Stat(delayedHAMarkerFile); err == nil {
+		return ceohelpers.DelayedHAScalingStrategy, nil
+	}
+	// Bootstrap in place when `bootstrapInPlace` root key exists in the install-config.
+	if _, found := installConfig["bootstrapInPlace"]; found == true {
```

Comment:
didn't you wanted to validate control plane replica == 1 ?

Reply:
is this necessary? it probably is better to be safe will update.